ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/platform.schema.yml

```diff
@@ -13,6 +13,119 @@ keys:
             type: str
             convert_types:
               - int
+          mmu:
+            type: dict
+            description: |
+              Memory Management Unit settings.
+            keys:
+              active_profile:
+                type: str
+                description: |
+                  The queue profile to be applied to the platform.
+              queue_profiles:
+                type: list
+                primary_key: name
+                convert_types:
+                  - dict
+                items:
+                  type: dict
+                  keys:
+                    name:
+                      type: str
+                    multicast_queues:
+                      type: list
+                      primary_key: id
+                      convert_types:
+                        - dict
+                      items:
+                        type: dict
+                        keys:
+                          id:
+                            type: int
+                            convert_types:
+                              - str
+                            required: true
+                            min: 0
+                            max: 7
+                          unit:
```

Comment:
to make this more intuitive I would suggest to rename to `reserved_unit`

Reply:
I would expect this unit to also impact the unit of the thresholds, but I don't know. We should verify on EOS.